### PR TITLE
[21054] Add XML configuration for FlowControllerDescriptor

### PR DESCRIPTION
**Pull request by Mario-DL**
## Description

This PR adds the XML configuration for FlowControllerDescriptor.

Related implementation PR:
* eProsima/Fast-DDS#4837

## Contributor Checklist

- [X] Commit messages follow the project guidelines. 
- [X] Code snippets related to the added documentation have been provided.
- [X] Documentation tests pass locally.
- __NO__ Applicable backports have been included in the description.

## Reviewer Checklist

- [x] The PR has a milestone assigned.
- [x] The title and description correctly express the PR's purpose.
- [x] Check contributor checklist is correct.
- [x] CI passes without warnings or errors.

### CODE REVIEW

**Comment by Mario-DL** [comment]
@richiprosima please test_3 this

File: docs/fastdds/use_cases/large_data/large_data.rst [hunk 1]
```diff
@@ -258,8 +258,10 @@ Example configuration
 
    .. tab:: XML
 
-      There is currently no way of configuring flow controllers with XML.
-      This will be added in future releases of the product.
+      .. literalinclude:: /../code/XMLTester.xml
+         :language: xml
+         :start-after: <!-->CONF-QOS-FLOWCONTROLLER
+         :end-before: <!--><-->
```
[the remark this answers is not in the document]
Comment (Mario-DL) [reply docs/fastdds/use_cases/large_data/large_data.rst:264]: Totally agree, nice catch !

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Comment by JesusPoderoso** [comment]
@richiprosima please test_3 this

File: docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst [hunk 2]
```diff
@@ -548,6 +548,8 @@ There are two possible values (see |PublishModeQosPolicyKind-api|):
 * |ASYNCHRONOUS_PUBLISH_MODE-api|: An internal thread takes the responsibility of sending the data asynchronously.
   The write operation returns before the data is actually sent.
 
+Also, the |PublishModeQosPolicy::flow_ctrl_name-api| has to be set to the name of a valid :ref:`flow-controllers` descriptor name.
+
```
Comment (JesusPoderoso) [inline docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst:552]: Linter complaints about line too long
```suggestion
Also, the |PublishModeQosPolicy::flow_ctrl_name-api| has to be set to the name of a valid :ref:`flow-controllers` 
descriptor name.

```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

**Review by JesusPoderoso — APPROVED** [review]

**Comment by JesusPoderoso** [comment]
@richiprosima please test_3 this

**Pull request merged**